src/main/java/com/chekanova/imagetool/controller/ImageController.java

```diff
@@ -14,33 +14,24 @@
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
-import javax.imageio.ImageIO;
-import java.awt.image.BufferedImage;
-import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 
 @RestController
 @RequestMapping("/image")
 @RequiredArgsConstructor
 public class ImageController {
-    private static final String JPG = "jpg";
     private final ImageService imageService;
 
     @PostMapping(value = "/process",
             produces = MediaType.IMAGE_JPEG_VALUE)
```

Comment:
static import for MediaType.IMAGE_JPEG_VALUE

Reply:
fixed